src/index.html

```diff
@@ -7,12 +7,65 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    >
+
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family
+Space+Grotesk:wght@300..700&display=swap"
+    rel="stylesheet"
+    >
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Imac-Logo"
+        class="card__image"
+      >
+      <div class="card__content">
+        <h2
+          class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="card__title card__title--code">Product code: 195434</h3>
+
+        <div class="card__container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__container-reviews">Reviews: 5</div>
+        </div>
+
+        <div class="price">
+          <h2 class="price__label">Price:</h2>
+          <h1 class="price__amount">$2,199</h1>
+        </div>
```

Comment:
```suggestion
        <div class="card__price">
          <h2 class="card__price-label">Price:</h2>
          <h1 class="card__price-amount">$2,199</h1>
        </div>
```
these elements also should be part of the card block. They can be nested in each other and still be elements of the common block (card)